src/main/kotlin/com/wire/bots/polls/setup/HttpClient.kt

```diff
@@ -32,23 +25,22 @@ fun createHttpClient(meterRegistry: MeterRegistry) =
         }
     }
 
-/**
- * Debug logger for HTTP Requests.
- */
-private val Logger.Companion.DEBUG: Logger
-    get() = object : Logger, org.slf4j.Logger by createLogger("DebugHttpClient") {
-        override fun log(message: String) {
-            debug(message)
-        }
-    }
-
-
 /**
  * Trace logger for HTTP Requests.
+ *
+ * Logs request/response bodies, params and headers.
+ * Avoids logging lines containing sensitive data
  */
 private val Logger.Companion.TRACE: Logger
     get() = object : Logger, org.slf4j.Logger by createLogger("TraceHttpClient") {
         override fun log(message: String) {
+            for (blockedWord in blockedWordList) {
+                if (message.contains(blockedWord, ignoreCase = true)) {
+                    return
+                }
+            }
             trace(message)
         }
     }
+
+private val blockedWordList = listOf("Authorization", "token", "Bearer", "text")
```

Comment:
`"text"` seems a bit broad, don't we catch what we want to catch already with `Authorization`?

Reply:
Also do I understand correctly that this is an additional safety measure as a last line of defense, as the code you changed already in `Message.kt` would make sure that when printing the message we don't print the token?
Not saying this is wrong, I'm just trying to wrap my head around it